pkg/hubspot/client.go

```diff
@@ -92,6 +109,33 @@ func (c *Client) GetUsers(ctx context.Context, getUsersVars GetUsersVars) ([]Use
 	return userResponse.Results, "", annos, nil
 }
 
+// GetUsers returns all users for a single workspace.
+func (c *Client) IsDeactivatedUser(ctx context.Context, email string) (bool, error) {
+	queryParams := url.Values{}
+
+	queryParams.Add("archived", "true")
+	queryParams.Add("limit", "1")
```

Comment:
is this something we can maybe build a cache for by listing through all owners, so we don't call it once per user?

Reply:
I think it's possible to add UserIdIncludingInactive and Archived to the existing Users struct and fetch all users (both active and deactivated) in the user resource List().

Another option would be to have a GetDeactivatedUsers() function that is called in List(). That would require tweaking the page token parsing so that we paginate both API calls.